examples/saving_collection.py

```diff
@@ -1,11 +1,11 @@
 import os
 import sys
 
-sys.path.insert(0, os.getenv("BUILDTEST_ROOT"))
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Why is this line needed?

Reply:
when i remove the line i run into import error

```
ssi29@lsfportal1> python examples/moduletest.py
Traceback (most recent call last):
  File "examples/moduletest.py", line 6, in <module>
    from buildtest.modules.util import Module
ModuleNotFoundError: No module named 'buildtest.modules'
```

Pretty sure its the packaging or something else i couldnt figure out
```
ssi29@lsfportal1> pip show -f buildtest-framework
Name: buildtest-framework
Version: 0.7.7
Summary: HPC Application Testing Framework
Home-page: https://github.com/HPC-buildtest/buildtest-framework
Author: Shahzeb Siddiqui
Author-email: shahzebmsiddiqui@gmail.com
License: GPLv2
Location: /u/users/ssi29/.local/lib/python3.7/site-packages/buildtest_framework-0.7.7-py3.7.egg
Requires: argcomplete, PyYAML, distro, termcolor
Required-by:
Files:
Cannot locate installed-files.txt
```

Looking into my egg i dont have the updated files such as ``buildtest.modules``

```
ssi29@lsfportal1> ll /u/users/ssi29/.local/lib/python3.7/site-packages/buildtest_framework-0.7.7-py3.7.egg/buildtest/
total 24
drwxr-xr-x 3 ssi29 hks 4096 Feb 22 17:13 docgen
-rw-r--r-- 1 ssi29 hks   60 Feb 22 17:13 __init__.py
-rw-r--r-- 1 ssi29 hks  739 Feb 22 17:13 main.py
drwxr-xr-x 2 ssi29 hks 4096 Feb 22 17:13 __pycache__
drwxr-xr-x 2 ssi29 hks 4096 Feb 22 17:13 scripts
-rw-r--r-- 1 ssi29 hks 1964 Feb 22 17:13 settings.yml
drwxr-xr-x 6 ssi29 hks 4096 Feb 22 17:13 tools

```

Hence i added that line so that it works with my git repo. I know i am missing something in my environment

Reply:
Did you install it?

Reply:
Yeah I don't see modules - I don't think it's installed. Is there an __init__.py in the folder?